common/src/Microsoft.Azure.IIoT.Hub.Mock/src/Services/IoTHubServices.cs

```diff
@@ -535,6 +538,8 @@ public class EventMessage {
             public Message Message { get; set; }
             /// <summary> The output target for sending the given message. </summary>
             public string OutputName { get; set; }
+            /// <summary/>
+            public byte[] MessageData { get; set; }
```

Comment:
Seems like `MessageData` is used only for tests, it is referenced only in `PublisherIntegrationTestBase` class. Maybe we can instead try to get bytes from the `Message` property itself when we receive the message, via `Message.GetBytes()` ? Then we can remove `MessageData` field.

Reply:
Fixed by #1676.